pycm/pycm_obj.py

```diff
@@ -866,3 +866,25 @@ def to_array(self, normalized=False, one_vs_all=False, class_name=None):
             row = [table[key][i] for i in classes]
             array.append(row)
         return numpy.array(array)
+
+    def combine(self, other):
```

Comment:
**For error checking :**
1. Error Type : `pycmMatrixError`
2. Error Message : `The input type is considered to be pycm.ConfusionMatrix object but it's not!`
3. Error Variable Name : `COMBINE_TYPE_ERROR`